fdbserver/Knobs.cpp

```diff
@@ -704,6 +704,7 @@ void ServerKnobs::initialize(bool randomize, ClientKnobs* clientKnobs, bool isSi
 	init( FASTRESTORE_RATE_UPDATE_SECONDS,                       1.0 ); if( randomize && BUGGIFY ) { FASTRESTORE_RATE_UPDATE_SECONDS = deterministicRandom()->random01() < 0.5 ? 0.1 : 2;}
 
 	init( REDWOOD_DEFAULT_PAGE_SIZE,                            4096 );
+	init( REDWOOD_DEFAULT_EXTENT_PAGES,                          256 );
```

Comment:
Let's make this a size in bytes, because page size could vary but the extent size should probably not scale up with it.

Reply:
Sounds good. Reverted it back to using size in bytes as it was initially implemented.